scripts/generate-clients.mjs

```diff
@@ -50,6 +50,13 @@ kinobi.update(
         "spl-token-2022"
       ),
     },
+    {
+      account: "stakeHistory",
+      defaultValue: k.publicKeyValueNode(
+        "SysvarStakeHistory1111111111111111111111111",
+        "stake-history"
+      ),
+    },
```

Comment:
nit: Is there a naming convention in place for sysvars? For example, if it's the clock, do you put it as `"clock"` or `"sysvarClock"`? If it's the former, then this is perfect, but if not, we should keep it consistent with other naming conventions for sysvars

Reply:
I don't think we have a naming convention. Perhaps it would be a good idea to have the `sysvar_*` prefix to clearly indicate that it is a Sysvar account.